docs/quickstart.rst

```diff
@@ -237,7 +239,12 @@ backed up and that the ``prune`` command is keeping and deleting the correct bac
     then
         info "Backup, Prune and/or Compact finished with an error"
     fi
-
+    
+    if [ ${global_exit} -lt 1 ];
```

Comment:
guess this should rather check for rc == 0.

Reply:
Good point.